lib/mayaUsd/render/mayaToHydra/renderOverride.h

```diff
@@ -175,6 +184,11 @@ class MtohRenderOverride : public MHWRender::MRenderOverride {
     HdRprimCollection _selectionCollection{
         HdReprTokens->wire, HdReprSelector(HdReprTokens->wire)
     };
+    HdRprimCollection _pointSnappingCollection{
```

Comment:
Might be worth also hiding this one behind a `20210000` version guard, since it's only used there.

Reply:
Good point.